### ISSUE TITLE
Can't deposit a file into VTW Dev from Elements (ed.lib.vt.edu)

### ISSUE THREAD

**Issue by mello99** [VTUL/vtechworks#397]
Hi all, I tried to deposit a file from Elements (ed.lib.vt.edu) into VTW Dev, but Elements keeps refreshing the page whenever I try to upload a file. So, in a nutshell, I can't actually upload a file to be deposited into VTW DEV from the ed.lib.vt.edu Elements server :(. That's the one remaining test I have. Thanks for your help, and take care.

**Comment by keithgee** [comment]
I'm not sure who thought to test this, but it was a good idea. I think the problem was related to the security changes on vtechworks.dev.
I was able to duplicate this and also couldn't deposit; Elements refreshed the page when I uploaded a file, but didn't give an error message.

On ed.lib, I have access to the Elements modules control panel. Under "Module Admin -> Repository Connection -> Edit the Existing Connection", I changed the "Service base URL" from http to https.

After the change, I was successfully able to deposit. 
The setting on efarqa.lib and efars.provost is also at http. I've left it because it this will help verify it was the cause of the problem, once vtechworks.pprd and vtechworks production are also updated with new security. @amandafrench has access to change this setting to https if I am vacationing when the change is made.

*\* Closing
